First, I noticed that I had left one sample out of the SRA list that I have been using, so I went back and ran the sra download, trimming, and alignment for that sample so I could include it in the differential exprtession comparison.

Counting
For counting, I used htseq-count with the command in a for loop of "htseq-count --idattr=ID -c counts/${sra}_count.csv align_outputs/${sra}_aligned.bam a_thaliana_annotation.gff". I used idattr=ID because some lines of the gff did not have gene_id, but everything had an ID. Also, despite me saying to export to csv, it still had the same formatting as the txt, which in some way was nice because it easily imported to DESeq2. I ran htseq-count as stranded since most Illumina reads now are stranded, although they didn't specify in the pulication, so I also ran unstranded as a backup. 

In DESeq2, I used the R code provided in lab, modified to follow my paths and file names, and using 'condition' of 'control' or 'drought' instead of the different genotypes. I ended up with 2 DE "genes" and 38798 notDE "genes", but I realize now that since I used ID, that's actually just gene segments and doesn't really tell me much. Now that all the code is set up, I will be going back to get a gff of genes only, and update to Araport 11 since I also realized I'm on TAIR10, and then will rerun the alignment with these updated files, which will affect the final DE results.