1. Trimming and fastqc on the fastq files
For trimming the sequences, I chose to use trimmomatic since it conveniently comes preloaded with the adapter sequences (which in this case was the TruSeq2), seemed to have better documentation, and is allegedley better than cutadapt for paired end reads, which mine are. I also included the leading and trailing quality trimming, especially since some of mine had some low quality reads towards the end of the reads (although not quite as important in RNAseq so I stuck with the recommended cutoff of 3). In the script, I made a list of my SRAs and then made a for loop, and then used another for loop to then run fastqc on each. I viewed the html outputs and found that all adapter content was gone, and the quality of the end of the reads was significantly increased to phred score >20 for pretty much everything. Some samples did lose several millions of reads through this process depending on quality and adapter content.
